CMake/fletch-tarballs.cmake

```diff
@@ -361,10 +361,21 @@ if(NOT WIN32)
 endif()
 
 # Protobuf
+
+set(Protobuf_SELECT_VERSION "2.5.0" CACHE STRING "Select the  version of ProtoBuf to build.")
+set_property(CACHE Protobuf_SELECT_VERSION PROPERTY STRINGS "2.5.0" "3.4.1")
```

Comment:
Should this set_property call go inside the if NOT WIN32? It seems like, as is, on Windows you would see options for a protobuf version though you can't build it.

Reply:
That seems reasonable. I didn't quite realize protobuf didn't build on windows. How does Caffe work on windows in this case? I thought it needed protobuf. 

After googling it appears protobuf can build on mingw. Should we make this a `NOT WIN32 OR MINGW` condition instead?